lib/v1/blogger/gratipay.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+This file is placed into Public Domain by
```

Comment:
Done. Because all grtp.co is Public Domain/CC0/UNLICENSE, it makes little sense to bear this note here. =)